dev/com.ibm.ws.org.apache.cxf.cxf.core.3.2/src/org/apache/cxf/message/MessageImpl.java

```diff
@@ -1385,5 +1388,24 @@ public Object merge(String key, Object value, BiFunction<? super Object, ? super
     public String[] getPropertyNames() {
         return propertyNames;
     }
+
+    @FFDCIgnore(ConcurrentModificationException.class)
+    @Override
+    public String toString() {
+        StringBuilder sb = new StringBuilder();
+        sb.append('{');
+        for (int i=0; i<TOTAL; i++) {
+            if (propertyValues[i] != null) {
```

Comment:
propertyValues can be null, so this should be "if (propertyValues[i] != NOT_FOUND) {"